### PR TITLE
fix:  If user look at the activity of the selected account, then in the third step the view will break

### LINKED ISSUE #628: If user look at the activity of the selected account, then in the third step the view will break
If user look at the activity of the selected account, then in the third step the view will break. Reproducible on product: 
account: ak_11111111111111111111111111111111273Yts 

https://github.com/aeternity/aescan/assets/69896204/3d5d2b62-129b-4b3d-8f34-662318b4e153

err

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #628
related https://github.com/aeternity/aescan/issues/362

## Demo

https://github.com/aeternity/aescan/assets/15363559/f326a9a7-446f-4e5c-b087-b984d195faa9

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add aex141 transfer event activity component

### CODE REVIEW

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
It seems to be fixed on testnet, but mainnet isn't working for me. I got 404 error in Chrome and Firefox. 

@janmichek please check it.

**Comment by janmichek** [comment]
> It seems to be fixed on testnet, but mainnet isn't working for me. I got 404 error in Chrome and Firefox.
> 
> @janmichek please check it.

Stange, I can access both deployments.

**Comment by janmichek** [comment]
I am not

> It seems to be fixed on testnet, but mainnet isn't working for me. I got 404 error in Chrome and Firefox.
> 
> @janmichek please check it.

I am consulting this infrastructure issue with Petar, it's not super reliable for sure.

**Review by Liubov-crypto — APPROVED** [review]
Fixed, LGTM

File: src/components/AccountActivityDataCellAex141TransferEvent.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+  <value-hash-ellipsed
+    :hash="activityPayload.contractId"
+    :link-to="`/contracts/${activityPayload.contractId}`"/>
```
Comment (michele-franchi) [inline src/components/AccountActivityDataCellAex141TransferEvent.vue:4]:
```suggestion
  <value-hash-ellipsed
    :hash="activityPayload.recipientId"
    :link-to="`/accounts/${activityPayload.recipientId}`"/>
```

Should be the recipient, it's "Transferred to".

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I would add the recipient address instead of the AEX-141 smart contract.

File: src/components/AccountActivityDataCellAex141TransferEvent.vue (hunk 1 above)
Comment (janmichek) [reply src/components/AccountActivityDataCellAex141TransferEvent.vue]: Make sense, fixed

**Review by michele-franchi — APPROVED** [review]
Looks good

**Pull request merged**